angular/package.json

```diff
@@ -10,10 +10,11 @@
     "lint": "ng lint",
     "coverage": "jest --coverage --config ./projects/spark-angular/jest.config.js",
     "docs:json": "compodoc --disableLifeCycleHooks -p ./tsconfig.json -e json -d .",
+    "build-styles": "cd '../styles/' && node build.js",
     "set-version": "PACKAGE_VERSION=$(node -p -e \"require('./projects/spark-angular/package.json').version\") && replace '(version).*' \"version: '$PACKAGE_VERSION'\" ./projects/spark-angular/src/environment/environment.ts",
-    "sassdocmodifiers": "sassdoc '../html/**/_*.scss' --parse > ../src/data/sass-modifiers.json",
+    "sassdocmodifiers": "sassdoc '../styles/**/_*.scss' --parse > ../src/data/sass-modifiers.json",
     "prebuild:storybook": "npm run sassdocmodifiers && npm run test:generate-output && npm run docs:json",
-    "storybook": "npm run prebuild:storybook && start-storybook -p 8006 -s ../storybook-utilities/favicon",
+    "storybook": "npm run build-styles && npm run prebuild:storybook && start-storybook -p 8006 -s ../storybook-utilities/favicon",
```

Comment:
Add styles build cmd to ng sb and it run when we launch sb so the vars are there in place.